dask/array/tests/test_linalg.py

```diff
@@ -520,13 +520,17 @@ def test_svd_compressed_shapes(m, n, k, chunks):
     assert v.shape == (r, n)
 
 
-def test_svd_compressed_compute():
+@pytest.mark.parametrize("iterator", [("none", 0), ("power", 1), ("QR", 1)])
+def test_svd_compressed_compute(iterator):
```

Comment:
Sorry @RogerMoens, I missed on our call that this is `test_svd_compressed_compute` and not `test_svd_compressed`.  I think we should probably have a test that validates the results of the QR iteration too.  

Does your updated `test_svd_compressed` function pass if you make the iteration type a parameter?

Reply:
@eric-czech: Yeah I see, I haven't updated the `test_svd_compressed`, that should be done. The reason, why I didn't was because that function also needed updates, which are accepted. The reason was more that it would be colliding, but probably GitHub has some fancy functionalities for that in place.

Do you want me to suggest those here for `test_svd_compressed`?

Reply:
Now that those changes are in can you pull them down on this branch?  I think this is ready to go once there's a test on the output of the QR iterator.

Reply:
Will have a look at it now, to fix this.